src/components/course/data/tests/hooks.test.jsx

```diff
@@ -1688,37 +1685,52 @@ describe('useCourseListPrice', () => {
     );
     expect(result.current).toEqual(mockListPrice);
   });
-  it('should not return the list price if one doesnt, first fallback, firstEnrollablePaidSeatPrice', () => {
+  it('should not return the list price if one doesnt, first fallback, fixed_price_usd', () => {
     const updatedListPrice = undefined;
     useCourseRedemptionEligibility.mockReturnValue({ data: { listPrice: updatedListPrice } });
     useCourseMetadata.mockReturnValue(updatedListPrice || getCoursePrice(baseCourseMetadataValue));
     const { result } = renderHook(
       () => useCourseListPrice(),
       { wrapper: Wrapper },
     );
-    expect(result.current).toEqual(baseCourseMetadataValue.activeCourseRun.firstEnrollablePaidSeatPrice);
+    expect(result.current).toEqual([baseCourseMetadataValue.activeCourseRun.fixedPriceUsd]);
   });
-  it('should not return the list price if one doesnt, second fallback, entitlements', () => {
+  it('should not return the list price if one doesnt, second fallback, firstEnrollablePaidSeatPrice', () => {
     const updatedListPrice = undefined;
     useCourseRedemptionEligibility.mockReturnValue({ data: { listPrice: updatedListPrice } });
+    delete baseCourseMetadataValue.activeCourseRun.fixedPriceUsd;
+    useCourseMetadata.mockReturnValue(updatedListPrice || getCoursePrice(baseCourseMetadataValue));
+    const { result } = renderHook(
+      () => useCourseListPrice(),
+      { wrapper: Wrapper },
+    );
+    expect(result.current).toEqual([baseCourseMetadataValue.activeCourseRun.firstEnrollablePaidSeatPrice]);
+  });
+  it('should not return the list price if one doesnt, third fallback, entitlements', () => {
```

Comment:
[sanity check] With the changed mock return values for `useCourseMetadata`, ensure these tests are actually testing the correct things (e.g., no passing as a red herring, etc.).